src/models/library_with_filters.rs

```diff
@@ -60,12 +60,24 @@ pub enum Sort {
     TimesWatched,
```

Comment:
There is already an enum for the filters called `Sort`, you should add `NotWatched` here